app/controllers/api/v1/merchants/revenue_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::Merchants::RevenueController < ApplicationController
+  def index
+    render json: Merchant.most_revenue(params[:quantity])
+  end
+
+  def show
+    merchant = Merchant.find(params[:id])
```

Comment:
Do we want to put this in `app/controllers/api/v1/merchants/revenue_by_date_controller.rb` instead?